packages/forms/src/UIForm/fields/index.js

```diff
@@ -5,7 +5,7 @@ import Comparator from './Comparator';
 import Datalist from './Datalist';
 import * as Date from './Date';
 import Enumeration from './Enumeration';
-import FieldTemplate from './FieldTemplate';
+import FieldTemplate, { TextModeTemplate } from './FieldTemplate';
```

Comment:
This import / export does not seems to be useful but i just want to be consistant